doc/manual/manual.tex

```diff
@@ -8485,23 +8488,249 @@ \subsubsection{The van Keken thermochemical composition benchmark}
+        zero contour $\tilde{\psi}^8 \, (t_{end})$ is bright green.
+        Compare with the fourth image on the right in Figure~\ref{fig:vk-1}.
+    }
+    \label{fig:VOF_van_Keken-02}
+\end{figure}
+
+Since, as noted above, when the interface is discontinuous, the van Keken problem is a version 
+of the Rayleigh-Taylor problem it is unstable to perturbations of all 
+wavelengths,\footnote{This is true whether the two fluids have the same viscosity or different 
+viscosities.} (e.g. see~\cite{SC:1961}), it is therefore, extremely sensitive to the initial 
+conditions.
+Thus, rather than use the standard initialization procedure for the VOF method, which consists 
+of integrating the initial interface over each multi-material cell in which one material or 
+fluid has the value of $1$ and the other has the value of $0$; i.e., a characteristic function 
+representation of the interface~\cite{JMR:2019,JMR-EGP:2019}, instead we compute the initial volume fractions using a signed distance function $\phi$ as follows.
+
+First we create the function $\phi$, which has the following two properties: 1) it is positive 
+in the region that contains one of the fluids, which we will refer to as fluid~1, and negative 
+in the complement of this region, which we will refer to as fluid~2, and 2) at each point in 
+the domain the magnitude $| \phi |$ of $\phi$  is the distance to the boundary between the 
+two fluids or materials.
+In the computations shown here, we use an approximation $\tilde{\phi}$ to $\phi$ such that   
+difference between $\tilde{\phi}$ and $\phi$ is small enough for the purposes of making the 
+computations shown here of high-quality.
+The primary advantage of choosing this particular initialization algorithm is that it allows us 
+to more accurately reproduce the initial condition on a sub-grid scale than would otherwise be 
+possible on the coarser grid on which we compute the time evolution of the interface. 
+
+\vspace{06pt}
+
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/vof-van-keken/main.part.prm.out}
+
+The relevant sections of the parameter file for this type of initialization of the VOF method 
+appears immediately above.
+In particular, the combination of \texttt{Number of initialization samples} with the 
+\texttt{level set} initialization type indicates that our initialization will consist of 
+dividing each grid cell into $16 \times 16$ subcells and the distance to the given initial 
+interface $f(x)$, provided in \texttt{Function expression}, is computed in each of the 256 
+subcells.
+We then use this information to compute a piecewise linear interface approximation to $f(x)$.
+The volume fraction in each subcell is then found in the manner described 
+in~\cite{JMR:2019,JMR-EGP:2019}.
+This initialization procedure provides a much finer and thus, more accurate, initial condition 
+than the standard VOF initialization procedure described above.
+
+% AMR section, currently unused so not included
```

Comment:
Is the plan to remove the following lines from the cookbook?

Reply:
That means you have a parameter file with AMR and VoF that crashes? Please open an issue about that and post the input file there so that others who encounter this crash will see that it is a known issue.

Reply:
@class4kayaker Is it possible to run this now and see if it works? Given that deal.ii is now v. 9.2.0, perhaps it will work?